src/main/java/org/gridsuite/filter/server/FilterService.java

```diff
@@ -65,41 +65,17 @@ public class FilterService {
     private final FilterService self;
 
     public FilterService(final ScriptFilterRepository scriptFiltersRepository,
-                         final LineFilterRepository lineFilterRepository,
-                         final GeneratorFilterRepository generatorFilterRepository,
-                         final LoadFilterRepository loadFilterRepository,
-                         final ShuntCompensatorFilterRepository shuntCompensatorFilterRepository,
-                         final StaticVarCompensatorFilterRepository staticVarCompensatorFilterRepository,
-                         final BatteryFilterRepository batteryFilterRepository,
                          final BusBarSectionFilterRepository busBarSectionFilterRepository,
```

Comment:
to be clean ?

Reply:
we will do it in another PR as said Christelle and Thibault